### PR TITLE
Add support for Node Type 42 (Outdoor Probe/Thermocouple)

### PR DESCRIPTION
**Pull request by mayermd**

### CODE REVIEW

File: profile/nls/en_us.txt [hunk 1]
```diff
@@ -41,8 +41,10 @@ ND-wTag13F-NAME = Motion Sensor Tag (13-bit temperature+humidity) (13) (F)
 ND-wTag21F-NAME = Motion Sensor Tag Pro (21) (F)
 ND-wTag26F-NAME = Motion Sensor Tag Pro ALS (Ambient Light Sensor) (26) (F)
 ND-wTag32F-NAME = Water/Soil moisture sensor (32) (F)
+ND-wTag42F-NAME = Outdoor Probe/Thermocouple (42) (F)
 ND-wTag52F-NAME = Door/window (reed) KumoSensor (52) (F)
 ND-wTag62F-NAME = Kumostat/Nest Thremostat (62) (F)
+ND-wTag62F-NAME = Kumostat/Nest Thremostat (62) (F)
```
Comment (jimboca) [inline profile/nls/en_us.txt]: Why was this line duplicated?

Comment (mayermd) [reply profile/nls/en_us.txt]: I'm sorry.  That was a copy paste error on my part.  That duplicate line can be replaced.  Please see the screenshot of the working node below.

screen shot 2018-03-14 at 6 38 53 am

**Comment by mayermd** [comment]
I have cleaned up the offending line and updated the README.  Let me know if you see any other issues.  Sorry for the mistakes.

**Comment by jimboca** [comment]
Thanks for the changes. I will run with this on my host and make sure no issues and release.

**Comment by jimboca** [comment]
All released, thanks again.

**Pull request merged**